README.md

```diff
@@ -1,96 +1,35 @@
-# A Question and Answer Bot Using Amazon Lex and Amazon Alexa
-
-> Build a chat bot to answer questions.
+# QnABot on AWS
 
 ## Overview
+QnABot on AWS is a multi-channel, multi-language conversational interface (chatbot) that responds to your customer’s questions, answers, and feedback. It allows you to deploy a fully functional chatbot across multiple channels including chat, voice, SMS, and Amazon Alexa. The solution’s content management environment, and contact center integration wizard allow you to set up and customize an environment that provides the following benefits:
 
-This repository contains code for the QnABot, described in the AWS AI blog post [“Creating a Question and Answer Bot with Amazon Lex and Amazon Alexa”](https://aws.amazon.com/blogs/ai/creating-a-question-and-answer-bot-with-amazon-lex-and-amazon-alexa/).
-
-Refer to [Getting Started](#getting-started) to launch your own QnABot, or [Automated Deployment](https://docs.aws.amazon.com/solutions/latest/qnabot-on-aws/automated-deployment.html) for detailed implementation instructions.
-
-The 5.1.0 release of QnABot provides support for a new mechanism where a Question can set a Lex session attribute.
-
-In prior versions, the use of handlebars in an answer could set a session attribute. For example, the following
-
-"{{setSessionAttr ‘attributeName' ‘attributeValue’}}""
-
-will set the an attribute called “attributeName” to the value “attributeValue”. The drawback in this approach is that the attribute value could not itself contain nested handle bars.
-
-In 5.1.0, a Question can now define a set of name/value pairs to set as session attributes when the answer is returned. You’ll see a new field to set a name/value pair plus an “Add” button to define additional name/value pairs and a button to delete a name/value pair. The Value field can contain handlebar statements as well.
-
-The attribute name can be a simple name like “myAttribute” or a complex name like “myAttribute.subAttribute”. Using the dot notation an attribute several levels deep can be set. Note that one can overwrite reserved session attribute names. Avoid using “appContext” or “qnabotcontext” as attribute names. Setting these may have adverse effects on the system.
-
-Note, the handlebar syntax still works just as it did before. The question configuration just has a new option.
-
-## Architecture Overview
-
-![Architecture](docs/architecture.png)
```

Comment:
Any reason removing the `Architecture Overview`  section?, all our solutions should display the solution's architecture in the root `README.md` file.